lib/machines/locales-machine.mdx

```diff
@@ -0,0 +1 @@
+# localesMachine
```

Comment:
The machine looks good! Please add a description, similar to the others, explaining it.

Reply:
Ok

Reply:
It's done. But how tags like <State></State> works.

Reply:
I got the meaning of tags !